lib/router/src/utils.ts

```diff
@@ -1,43 +1,16 @@
 import qs from 'qs';
 import memoize from 'memoizerific';
-import startCase from 'lodash/startCase';
+
+// FIXME: Remove in SB 6.0
+export { toId, parseKind, sanitize } from '@storybook/csf';
```

Comment:
@shilman Why did you have to remove `storyNameFromExport` and other exports that are present in [`5.2`](https://github.com/storybookjs/storybook/blob/master/lib/router/src/utils.ts)?

Reply:
@Hypnosphi That was accidental. I'll add `storyNameFromExport` back. That's the only one, right?

Reply:
That's the only one I used =) And probably the only one at all